float32 myFloat
int32 myInt
float32[] myArray